src/test/regress/sql/pg13.sql

```diff
@@ -115,5 +115,18 @@ EXPLAIN (ANALYZE TRUE, WAL TRUE, COSTS FALSE, SUMMARY FALSE, BUFFERS FALSE, TIMI
 WITH cte_1 AS (INSERT INTO test_wal VALUES(6,66),(7,77),(8,88) RETURNING *)
 SELECT * FROM cte_1;
 
+CREATE TABLE distributed_table(a int, b int);
+SELECT create_distributed_table('distributed_table', 'a');
+
+CREATE TABLE reference_table(a int, c int, b int);
+SELECT create_reference_table('reference_table');
+
+SELECT distributed_table.* from distributed_table JOIN reference_table ON (true);
```

Comment:
what does that first select test ?

Reply:
You mean why do we have `SELECT distributed_table.* from distributed_table JOIN reference_table ON (true);` here?

We test both that it works correctly in normal case and also that it still works correctly after dropping a column. For example it might be that it works correct only after dropping a column.

Reply:
I believe we have lots of tests to test the behavior before drop column but I'm fine with keeping both